src/lib/redux-cozy-api.js

```diff
@@ -286,6 +286,13 @@ export const downloadArchive = async (notSecureFilename, fileIds) => {
   forceFileDownload(fullpath, filename + '.zip')
 }
 
+export const downloadFile = async (file) => {
+  const response = await cozy.client.files.downloadById(file.id)
+  const blob = await response.blob()
+  const filename = file.name
+  forceFileDownload(window.URL.createObjectURL(blob), filename)
+}
```

Comment:
As this function is not related to redux, I think it shouldn't be sited here. I asked @goldoraf his advice. But I suggest you let it there for now, and we will move it away later.